docs/introduction/infrastructure.md

```diff
@@ -1,161 +1,104 @@
 # Computational Infrastructure
 
-## Introduction
-One way of understanding the Inductiva API is to see it as an abstraction layer that
-allows you to use computational resources made available by different providers in a
-seamless and uniform fashion. 
+One way to understand the Inductiva API is to see it as an intermediary between 
+users and the complex landscape of computational resources. In this reference, you 
+will learn how the Inductiva API manages your computational workload, and then you'll
+explore the various computational options currently available to you, and additional resources set to be introduced in future updates.
 
-On the server side, Inductiva takes a computational load issued by the user, i.e.,
-one or several simulations, and routes it to a computational resource that the users
-has access to, and takes care of all the orchestration required for running the load
-and returning the results back to the user. 
+## Computational Resource Management
 
-The computational resources effectively used to process that load can potentially be
-provided by a number of different players, and it is Inductiva’s role to facilitate
-the access to those resources, and even help you find the best one for your particular
-case (e.g. by price or by performance). 
+The Inductiva API acts as an abstraction layer that enables you to access a wide 
+array of computational resources across different providers through a unified 
+interface called the [Command Line Interface (CLI)](). The API's key role is to 
+assist you in identifying the resources best suited to your needs, whether you're prioritizing cost-efficiency or performance.
 
-Inductiva API is being designed to allow using compute resources such as:
+These resources could be from cloud providers, bare-metal hardware rentals, standard 
+high-performance computing (HPC) solutions commonly used in academia, or even on-premise 
+hardware for those with their own computing infrastructure. The API serves as a unifying interface atop all these varied resources, allowing you to tap into computational solutions of varying scales, prices, and performance levels through straightforward Python 
+scripting from your laptop.
```

Comment:
This last line seems to be analogous to what we want to say in the first paragraph you have added to this section. Try to merge both at the top, and then we can elaborate on the functionalities permitted.